src/utils/conversation.js

```diff
@@ -27,6 +27,30 @@ export function hasUnreadMentions(conversation) {
 			&& (conversation.type === CONVERSATION.TYPE.ONE_TO_ONE || conversation.type === CONVERSATION.TYPE.ONE_TO_ONE_FORMER))
 }
 
+/**
+ * check if the conversation is archived
+ *
+ * @param {object} conversation conversation object
+ * @return {boolean}
+ */
+export function isArchived(conversation) {
+	return conversation.isArchived
+}
+
+/**
+ * check if the conversation passes default validation:
+ * - non-archived
+ * - archived, but have an unread message with notification triggered
+ *
+ * @param {object} conversation conversation object
+ * @return {boolean}
+ */
+export function isDefault(conversation) {
+	return !isArchived(conversation)
+		|| (conversation.notificationLevel === PARTICIPANT.NOTIFY.ALWAYS && hasUnreadMessages(conversation))
+		|| (conversation.notificationLevel === PARTICIPANT.NOTIFY.MENTION && hasUnreadMentions(conversation))
```

Comment:
notification level can be `ALWAYS` or (`DEFAULT` and it is 1-1 conv) for unread messages
can be `ALWAYS` or `DEFAULT` or `MENTION` for unread mentions

Reply:
I think we discussed to not make it dependant on the level for now? Direct mention will show it again, regardless of notification setting ?

Reply:
DEFAULT is backend-only for unassigned yet conversation (by default, it's ALWAYS for 1-1 chats and MENTION for group chats, so we don't see and don't consider it in frontend

See `hasUnreadMessages()` and `hasUnreadMentions()` methods above, they should include these cases correctly